core/src/main/java/io/questdb/cairo/map/FastMap.java

```diff
@@ -333,22 +333,63 @@ private FastMapValue asNew(BaseKey keyWriter, int index, int hashCode, FastMapVa
         kPos = (kPos + 7) & ~0x7;
         setOffset(offsets, index, keyWriter.startAddress - heapStart);
         setHashCode(offsets, index, hashCode);
-        if (--free == 0) {
```

Comment:
Shouldn't we change the default load factor for `FastMap` to 0.8 or even 0.9? To me, the main benefit of using Robin Hood hashing is being able to fit more entries into less memory and that should improve performance in certain scenarios.

Reply:
good idea, I can test it.